lib/buildsystems/autotools.rb

```diff
@@ -1,10 +1,27 @@
 require 'fileutils'
 require 'package'
+require 'require_gem'
 
 class Autotools < Package
   property :configure_options, :pre_configure_options, :configure_build_extras, :configure_install_extras
 
   def self.build
+    puts "Additional #{superclass.to_s.capitalize} options being used:".orange
+    method_list = methods.grep(/configure_/).delete_if { |i| send(i).blank? }
+    require_gem 'method_source'
+    method_blocks = []
+    method_strings = []
+    method_list.sort.each do |method|
+      @method_info = send method
+      if @method_info.is_a? String
+        method_strings << "#{method}: #{@method_info}".orange
+      else
+        method_blocks << @method_info.source.to_s.orange
+      end
+    end
+    puts method_strings
+    puts method_blocks
```

Comment:
Is this something we could black box since it is used in all the buildsystem build methods?

Reply:
Entirely possible. Maybe @Zopolis4 would be up for doing that during one of her simplification spurts?